packages/admin-e2e-tests/src/specs/activate-and-setup/complete-onboarding-wizard.ts

```diff
@@ -349,8 +351,175 @@ const testDifferentStoreCurrenciesWCPay = () => {
 	} );
 };
 
+const testSubscriptionsInclusion = () => {
+	describe( 'A non-US store will not see the Subscriptions inclusion', () => {
+		const profileWizard = new OnboardingWizard( page );
+		const login = new Login( page );
+
+		beforeAll( async () => {
+			await login.login();
+		} );
+
+		it( 'can complete the store details section', async () => {
+			await profileWizard.navigate();
+			await profileWizard.storeDetails.completeStoreDetailsSection( {
+				countryRegionSubstring: 'fran',
+				countryRegionSelector: 'FR',
+				countryRegion: 'France',
+			} );
+
+			// Wait for "Continue" button to become active
+			await profileWizard.continue();
+
+			// Wait for usage tracking pop-up window to appear
+			await profileWizard.optionallySelectUsageTracking();
+		} );
+
+		it( 'can complete the product types section, Subscriptions copy is not visible', async () => {
+			// Query for the industries checkboxes
+			await profileWizard.industry.isDisplayed();
+			await profileWizard.industry.uncheckIndustries();
+			await profileWizard.industry.selectIndustry( 'Health and beauty' );
+			await profileWizard.continue();
+			await profileWizard.productTypes.isDisplayed( 7 );
+			await profileWizard.productTypes.uncheckProducts();
+			await profileWizard.productTypes.selectProduct( 'Subscriptions' );
+			await expect( page ).not.toMatchElement( 'p', {
+				text:
+					'The following extensions will be added to your site for free: WooCommerce Payments. An account is required to use this feature.',
+			} );
+
+			await profileWizard.continue();
+			await page.waitForNavigation( { waitUntil: 'networkidle0' } );
+		} );
+
+		it( 'can complete the business details tab', async () => {
+			await profileWizard.business.isDisplayed();
+
+			await profileWizard.business.selectProductNumber(
+				config.get( 'onboardingwizard.numberofproducts' )
+			);
+			await profileWizard.business.selectCurrentlySelling(
+				config.get( 'onboardingwizard.sellingelsewhere' )
+			);
+
+			await profileWizard.continue();
+		} );
+
+		it( 'cannot see the WooCommerce Payments extension after it has been installed', async () => {
```

Comment:
This expect here is expecting `WooCommerce Payments` to show up, but the test description say's otherwise?
I assume it should show `WooCommerce Payments` here for a non-us store.

Reply:
You're right, I fixed the description in commit 982450c